db/dcrpg/pgblockchain.go

```diff
@@ -6324,3 +6324,8 @@ func (pgb *ChainDB) MixedUtxosByHeight() (heights, utxoCountReg, utxoValueReg, u
 	return
 
 }
+
+// OldestTransaction queries the database for the data of the oldest address transaction
```

Comment:
Does not query the database though. Is that what you intend?

Reply:
I am just getting the oldest transaction of an address from the cache.